docs/excel/custom-functions-overview.md

```diff
@@ -28,109 +34,87 @@ In the cloned sample repo, you’ll see the following files:
 
 - **./src/customfunctions.js**, which contains the custom function code (see the simple code example above for the `ADD42` function).
 - **./config/customfunctions.json**, which contains the registration JSON that tells Excel about your custom function. Registration makes your custom functions appear in the list of available functions displayed when a user types in a cell.
-- **./index.html**, which provides a &lt;Script&gt; reference to the JS file. This file does not display UI in Excel.
```

Comment:
Got rid of it because I don't think that this is universally true across Custom Functions. On desktop you actually do get a taskpane that shows the content from the html file. The task pane doesn't appear on Excel Online. I'll fix to reflect that more accurately.